File: Day 1/PYTHON/day_1.py
import random 
import datetime
count_exact_answer = 0
count_wrong_answer = 0

for number in range(10):
	number1 = random.randrange(31, 61)
	number2 = random.randrange(1, 30)
	result = number1 - number2
	

	try:
		question = int(input(f"What is {number1} - {number2}: "))
		if question < 1:
			raise ValueError ("Invalid input brother!")
		if question == result:
			count_exact_answer+=1
			print("Good!")

		else:
			print("Wrong guess. Try once more")
			question = int(input(f"What is {number1} - {number2}: "))
			count_wrong_answer +=1
			if question:
				print("You no get brain!")
			
	except ValueError:
		print("Enter a valid input!")
		question = int(input(f"What is {number1} - {number2}: "))
		if question == result:
			print("Good!")
			continue

print(" ")
if count_exact_answer > count_wrong_answer:
		
	print(f"Good! You got {count_exact_answer} out of 10")

else:
	print(f"Oops! You lost {count_wrong_answer} out of 10")

	

File: Day 1/JAVA/Day1.java
import java.util.Scanner; 
import java.util.Random;
public class Day1{
public static void main(String...args){
Scanner input = new Scanner(System.in);
int count_exact_answer = 0;
int count_wrong_answer = 0;

	Random repeat = new Random();
	

	int toRun = 1;
	while(true){
	int number1 = repeat.nextInt(91)+34;
	int number2 = repeat.nextInt(33)+1;
	int result = number1 - number2;

	System.out.printf("What is %d - %d%n ", number1, number2 + toRun);
	int question = input.nextInt();
	toRun++;
	if (toRun > 10){
	break;
	}
	if (question < 1){
	
		System.out.println("Invalid input");
	}else{

		if (result == question){
			count_exact_answer+=1;
			System.out.println("Good! You won");
	
		}else{
			count_wrong_answer +=1;
			System.out.println("Try again! You lost");
		}

		if (count_exact_answer > count_wrong_answer){
		
			System.out.printf("Good! You won %d times\n", count_exact_answer);

		}else{
			System.out.printf("Oops! You lost %d times\n", count_wrong_answer);
		}
	}
	}
	}

}